### PR TITLE
Replace oil with canola oil for healthier living

### PR DESCRIPTION
**Pull request by newind27**

### CODE REVIEW

File: cake-recipe.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup canols oil
```
Comment (adnewind) [inline cake-recipe.txt]: canols or canola?

Comment (adnewind) [inline cake-recipe.txt]: canols or canola?

Comment (newind27) [inline cake-recipe.txt]: Sorry, it should be canola. will edit it.

**Comment by newind27** [comment]
Please check if this is healthier but with more oil. Ok for merging?

**Pull request merged**